src/renderer/components/settings-electron.tsx

```diff
@@ -383,7 +383,7 @@ export const ElectronSettings = observer(
           buttonProps.onClick = () => {
             isLocal
               ? appState.removeVersion(ver)
-              : appState.downloadVersion(ver);
+              : appState.downloadVersion(ver, { noActivate: true });
```

Comment:
_nit_

Negated logic like this is often harder to follow / easier to accidentally get wrong later on.

Would it be possible to make the opts `activate` instead, defaulting to true?